domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -490,4 +557,52 @@ class StoryProgressTestHelper @Inject constructor(
       timestamp
     )
   }
+
+  /**
+   * Marks one explorations in first test topic as completed played for a particular profile.
```

Comment:
Nit on wording: Marks one exploration in first...

Ditto below.

Reply:
Done.